CSharp_Net_module1_1_4_lab/CSharp_Net_module1_1_4_lab/Program.cs

```diff
@@ -0,0 +1,269 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Remoting.Metadata.W3cXsd2001;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharp_Net_module1_1_4_lab
+{
+    class Program
+    {
+        // 1) declare enum ComputerType
+
+        enum ComputerType
+        {
+            Desktop,
+            Laptop,
+            Server
+        }
+
+        // 2) declare struct Computer
+
+        struct Computer
+        {
+            public ComputerType type;
+            public int count;
+            public int CPUCores;
+            public float CPUFrequency;
+            public int memory;
+            public int storage;
+        }
+
+        static void Main(string[] args)
+        {
+            // 3) declare jagged array of computers size 4 (4 departments)
+
+            Computer[][] departments = new Computer[4][];
+
+            // 4) set the size of every array in jagged array (number of computers)
+
+            for (int i = 0; i < departments.GetLength(0); i++)
```

Comment:
Do you really use a jagged array? 
It looks like you have a multidimensional array. 
Recall the task.